specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/managedInstances.json

```diff
@@ -484,6 +438,111 @@
         "instancePoolId": {
           "description": "The Id of the instance pool this managed server belongs to.",
           "type": "string"
+        },
+        "maintenanceWindowSettings": {
+          "$ref": "#/definitions/MaintenanceWindowSettings",
+          "description": "Specifies maintenance window settings for a managed instance.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      }
+    },
+    "MaintenanceWindowSettings": {
+      "description": "The properties of managed instance maintenance window.",
+      "required": [
+        "frequency",
+        "customerTimeZone"
+      ],
+      "type": "object",
+      "properties": {
+        "timeOfUpgrade": {
+          "description": "Specifies time of upgrade for maintenance window of managed instance.",
+          "type": "string",
```

Comment:
what is the format of this value? good to clarify that in the description. (e.g. 12-hour AM/PM, 24-hour clock)